datasource/src/main/scala/database/SlickDatabase.scala

```diff
@@ -0,0 +1,58 @@
+package database
+
+import java.sql.Connection
+
+import com.typesafe.config.{Config, ConfigFactory}
+import liquibase.database.DatabaseFactory
+import liquibase.database.jvm.JdbcConnection
+import liquibase.resource.ClassLoaderResourceAccessor
+import liquibase.{Contexts, LabelExpression, Liquibase}
+import slick.basic.DatabaseConfig
+import slick.jdbc.JdbcProfile
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+import scala.util.Try
+
+class SlickDatabase(val config: Config = ConfigFactory.load()) extends SqlDatabase with AutoCloseable {
```

Comment:
Could you please specify what is the reason to use default parameter here? Is there any chance that somebody will generate config in the way other than ConfigFactory.load()?

Reply:
In order to have an ability to use different DB configs, we have to define this parameter here.
You will see how it will be used for tests soon.

It might looks like this `new SlickDatabase(ConfigFactory.load("test_configuration"))`